Week-2/index.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <title>Week 2 Assignment 1</title>
+</head>
+
+<body>
+    <h1>Lorem ipsum dolor sit amet consectetur adipisicing elit.</h1>
+
+    <div class="main-article">
+        <div>
+            <article class="core-article">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Non corrupti aut cum esse nesciunt itaque, quo
+                quibusdam
+                explicabo vel quia! Vero nam quaerat fuga ad deleniti repudiandae officiis atque ratione voluptatem
+                obcaecati,
+                temporibus possimus nulla necessitatibus?
+            </article>
+        </div>
+
+        <div>
+            <article class="core-article">
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Non corrupti aut cum esse nesciunt itaque, quo
+                quibusdam
+                explicabo vel quia! Vero nam quaerat fuga ad deleniti repudiandae officiis atque ratione voluptatem
+                obcaecati,
+                temporibus possimus nulla necessitatibus?
+            </article>
+        </div>
+
+        <div class='aeroplane'>
+            <img class="image-aero"
+                src="https://images.unsplash.com/photo-1529074963764-98f45c47344b?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8YWVyb3BsYW5lfGVufDB8fDB8fA%3D%3D&w=1000&q=80"
+                alt="">
+        </div>
+
+        <div>
+            <article class="core-article">
+                <b>Lorem ipsum dolor sit</b>
+                <br>
+                <br>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Non corrupti aut cum esse nesciunt itaque, quo
+                quibusdam
+                explicabo vel quia! Vero nam quaerat fuga ad deleniti repudiandae officiis atque ratione voluptatem
+            </article>
+        </div>
+
+        <div>
+            <article class="core-article">
+                <b>Lorem ipsum dolor sit</b>
+                <br>
+                <br>
+                Lorem ipsum dolor sit amet consectetur adipisicing elit. Non corrupti aut cum esse nesciunt itaque, quo
+                quibusdam
+                explicabo vel quia! Vero nam quaerat fuga ad deleniti repudiandae officiis atque ratione voluptatem
+            </article>
+        </div>
+
+    </div>
+
+
+    <form class="form-class">
```

Comment:
@relhanakshay form also seems not responsive. Can you check it out.

Reply:
<img width="326" alt="Screenshot 2022-07-30 at 10 58 50 AM" src="https://user-images.githubusercontent.com/9133593/181876214-893e1495-a20c-4d46-9a0a-91131504984c.png">

Reply:
I have checked that the form is already responsive.